adot/utils/sam/layer.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Are these two files in the `sam` folder related to the soak test?

Reply:
should move to `utils/common` folder, for enabling lambda insight.
Lambda insight actually is also a layer layer, we can enable it by:
```
./update.sh -a "arn:aws:lambda:us-west-2:580247275435:layer:LambdaInsightsExtension:10"
```

Reply:
Ah ok

Reply:
Sent https://github.com/open-telemetry/opentelemetry-lambda/pull/58

Reply:
Just realize that I can enable LambdaInsight in soak.py by python aws sdk directly, no need use awscli to enable LambdaInsight. 
Now we can run soak in one step only:
* console:
python3 soak.py -i 5 -t 1600 --endpoint=https://y1j79idv7e.execute-api.us-west-2.amazonaws.com/api/ -n otel-stack-function-1JV2MX5O73IYO --cpu-threshold=60 --memory-threshold=45
* docker:
docker run -v ~/.aws/:/root/.aws:ro soak -i 5 -t 1600 --endpoint=https://y1j79idv7e.execute-api.us-west-2.amazonaws.com/api/ --name=otel-stack-function-1JV2MX5O73IYO --cpu-threshold=60 --memory-threshold=45

Reply:
Nice! Can we remove these files then?

Reply:
`utils/sam` is removed.